jobs/ml-ops/terraform/images.tf

```diff
@@ -8,6 +8,7 @@ resource "docker_image" "inference" {
   name = "${scaleway_registry_namespace.main.endpoint}/inference:${var.image_version}"
   build {
     context = "${path.cwd}/../inference"
+    no_cache = true
```

Comment:
**question(clarification):** why are these `no_cache` needed?

Reply:
They are not necessary in fact. Removed.